src/pages/home/report/ReportActionsView.js

```diff
@@ -242,14 +246,17 @@ class ReportActionsView extends React.Component {
 
         this.updateSortedReportActions();
         return (
-            <InvertedFlatList
-                ref={el => this.actionListElement = el}
-                data={this.sortedReportActions}
-                renderItem={this.renderItem}
-                contentContainerStyle={[styles.chatContentScrollView]}
-                keyExtractor={item => `${item.action.sequenceNumber}`}
-                initialRowHeight={32}
-            />
+            <>
+                <AttachmentModal sortedReportActions={this.sortedReportActions} />
```

Comment:
Potentially, I wanted to have most of the changes in the AttachmentModal file which is why I simply passed the actions straight into the modal. If you want to filter it before and then pass it in, that's and easy chage.

Reply:
>I wanted to have most of the changes in the AttachmentModal file which is why I simply passed the actions straight into the modal

Cool, I think it should be done differently. The `AttachmentModal` should be concerned with attachments. It should not be concerned with all report actions or be responsible for filtering them and extracting the attachments inside them.

cc @chiragsalian just to make sure we are on the same page with this.

Reply:
Actually yeah i definitely agree with this. If we filter all attachments here and pass just `attachmentActions` it would be so much lighter on the logic in `AttachmentModal` since it won't have to filter through all report actions on every keypress.

Reply:
ok